### DISCUSSION TITLE
Is there a Quarto extension to create curriculum vitae in html and pdf?

### DISCUSSION THREAD

**Discussion by luifrancgom** [quarto-dev/quarto-cli#11704] [Q&A]
### Description

I want to create a curriculum vitae in html and pdf in Quarto. I search for different solutions like:

- [quarto-cv](https://github.com/schochastics/quarto-cv): It only includes templates in pdf
- [Publishing HTML and PDF versions of a Quarto CV without LaTex](https://www.cynthiahqy.com/posts/cv-html-pdf/): It is not flexible enough to add information in the header for pdf and html
- [vitae package](https://pkg.mitchelloharawild.com/vitae/) : For now, it does not have Quarto support https://github.com/mitchelloharawild/vitae/issues/215

Is there any Quarto extension to create a curriculum vitae in html and pdf that maybe someone knows? If it is not available how should I proceed?

I have not created a Quarto extension where I don't know if it is possible to transform at least one template pointed out in [quarto-cv](https://github.com/schochastics/quarto-cv) to  html. Any suggestions will be appreciated. For example I don't know if I should begin with [Creating Extensions](https://quarto.org/docs/extensions/creating.html) and [Custom formats](https://quarto.org/docs/extensions/formats.html)

Finally, in case that the only possibility is to create a custom format and taking into account what is mention in [FAQ Rmarkdown](https://quarto.org/docs/faq/rmarkdown.html#can-you-create-custom-formats-for-quarto-like-you-can-for-r-markdown) in relation to *... A major difference with custom output format in R Markdown is that Quarto Extension does not use R but Lua ...* what is a good source to begin to experiment with Lua.

**Comment by mcanouil** [comment]
You need to understand the format you are targeting in order to make an extension.
You should read both guides you've linked (it includes link to external resources).
Additionally, you should read <https://quarto.org/docs/journals/templates.html>.

There are two listings of extensions which are unlikely to be exhaustive:
- https://m.canouil.dev/quarto-extensions/ (it includes https://quarto.org/docs/extensions/)
- https://quarto.org/docs/extensions/

**Reply by luifrancgom** [reply]
Thank you Mickaël Canouil for the suggestions.
